src/events/message/messageCreate.ts

```diff
@@ -80,7 +80,7 @@ export default class extends Event {
 
 		try {
 			await command.run(context, options.args);
-		} catch (err: any) {
+		} catch (err: unknown) {
```

Comment:
```suggestion
		} catch (err: any) {
```
This needs to be `any` to fix the TS issue. Caught errors are already `unknown` by default, that's why it showed a TS problem.